File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int number;
    do
    {
        number = get_int("Height: ");
    }
    while ( number<1 || number>8);

    for (int i=0; i<number; i++){
        for (int j=number-i; j>1; j-=1)
             {printf(" ");}
        for (int j=-1; j<i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j=-1; j<i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py

def get_int(prompt):
    while True:
        try:
            height = int(input(prompt))
            if not ( 1 <= height <= 8 ):
                print("Invalid input.\n")
            else:
                return height
        except ValueError:
            print("Not an integer")

height = get_int("Height: ")


for i in range(1, height+1):
    # print("i=", i)
    # print("height=", height)
    space = height - i
    print(' '*space + "#"*i + '  ' + "#"*i)
